app/config/services/services.xml

```diff
@@ -351,5 +351,12 @@
 
         <!-- Storage -->
         <service id="AppBundle\Storage\ImageStorage" />
+
+        <!-- ParamConverter -->
+        <service id="AppBundle\Request\ParamConverter\DoctrineQueryStringConverter">
```

Comment:
1) I think `App\ParamConverter` as namespace will be enough 
2) Can you add Param in the class name `...StringConverter` => `...StringParamConverter`?

Reply:
Done.